back/lib/voting_method/voting.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module VotingMethod
+  class Voting < Base
+    def initialize(participation_context)
+      # Set the default voting term for the context
+      if participation_context.voting_term_singular_multiloc.blank? && participation_context.voting_term_plural_multiloc.blank?
+        participation_context.voting_term_singular_multiloc = CL2_SUPPORTED_LOCALES.index_with do |locale|
+          I18n.t('voting_method.default_voting_term_singular', locale: locale)
+        end
+        participation_context.voting_term_plural_multiloc = CL2_SUPPORTED_LOCALES.index_with do |locale|
+          I18n.t('voting_method.default_voting_term_plural', locale: locale)
+        end
+      end
+      super
+    end
+
+    def validate
```

Comment:
If the singular and plural voting terms are both blank, no error is raised. Is this desired behaviour?

Reply:
Yes it was, because if they are both blank it should use the default term